src/Providers/Dailymotion.php

```diff
@@ -59,6 +58,10 @@ public function get_thumbnail_data(): array {
 					)
 				);
 
+				if ( $response_body === null ) {
```

Comment:
I just to make sure we don't run into a different error. It looks like `json_decode` expects a string for the first argument. I'm guessing the root issue here is that `wp_remote_retrieve_body` returns nothing? Do you know if it is giving a string or a null? If it is returning null itself, maybe we move the if statement before the `json_decode`.